src/BitcoinNetworkEncoder.sol

```diff
@@ -0,0 +1,47 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.24;
+
+library BitcoinNetworkEncoder {
+
+    bytes constant BTC_BECH32_MAINNET_BYTES = hex"626331"; // prefix = bc1
+    bytes constant BTC_BECH32_TESTNET_BYTES = hex"746231"; // prefix = tb1
+    bytes constant BTC_BECH32_REGTEST_BYTES = hex"6263727431"; // prefix = bcrt1
+
+    string constant BTC_BECH32_MAINNET = 'bc';
+    string constant BTC_BECH32_TESTNET = 'tb';
+    string constant BTC_BECH32_REGTEST = 'brct';
```

Comment:
is it correct that first 3 constants contain 3 chars and last 3 only 2 chars?

Reply:
That is because it is just copy-paste of Aleksey's code into Mykola's code, keeping their codebases. That's the difference of their code styles :)